src/Contributor/ContributorsUpdatedFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Contributor;
+
+use CultuurNet\UDB3\Event\EventContributorsUpdated;
+use CultuurNet\UDB3\Iri\IriGeneratorInterface;
+use CultuurNet\UDB3\Organizer\OrganizerContributorsUpdated;
+use CultuurNet\UDB3\Place\PlaceContributorsUpdated;
+
+final class ContributorsUpdatedFactory
+{
+    private IriGeneratorInterface $eventContributorsUpdatedIriGenerator;
```

Comment:
The name can be tweaked, it is not the IRI generator of the update endpoint but of the get endpoint